bytecomp/lambda.ml

```diff
@@ -264,9 +264,25 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+type module_map = {
+  map :  (int * Ident.t) list;
```

Comment:
Well it was not clear, but an integer is not sufficient to identify a root with flambda. A root is a field in a symbol.
map would be `(int * symbol *Ident.t) list`
The methcache, like the other roots is a field in some symbol. In the flambda version it is simply a variable defined before everything else.